src/integration-tests/introspector/mii/models/domain1.yaml

```diff
@@ -0,0 +1,105 @@
+domainInfo:
```

Comment:
you need copyright and license statements in yaml files
unless there is some technical reason they cannot be here
@ddsharpe i assume WDT can handle comments, right? i believe i have WDT models with comments in them

Reply:
Fixed missing copyrights